src/components/TeamMemberTasks/components/TaskCompletedModal.jsx

```diff
@@ -38,38 +31,37 @@ const TaskCompletedModal = React.memo(props => {
     props.updateTask(task._id, updatedTask);
   };
 
-  useEffect(() => {
-    loadUserTasks(props.userId);
-  }, [props.userID, props.tasks]);
-
   return (
     <Modal isOpen={props.isOpen} toggle={() => props.popupClose()}>
       <ModalHeader toggle={() => props.popupClose()}>Mark as Done</ModalHeader>
-      <ModalBody>
-        <p>Are you sure you want to mark this task as done?</p>
-        <ModalFooter>
-          <Button
-            color="primary"
-            onClick={() => {
-              removeTaskFromUser(props.task);
-              props.setClickedToShowModal(false);
-              props.setCurrentUserId('');
-              closePopup();
-            }}
-          >
-            Confirm
-          </Button>
-          <Button
-            onClick={() => {
-              props.setClickedToShowModal(false);
-              props.setCurrentUserId('');
-              closePopup();
-            }}
-          >
-            Cancel
-          </Button>
-        </ModalFooter>
-      </ModalBody>
+      {isLoadingTask ? (
+        <ModalBody>
+          <p>Loading...</p>
+        </ModalBody>
+      ) : (
+        <ModalBody>
+          <p>Are you sure you want to mark this task as done?</p>
+          <ModalFooter>
+            <Button
+              color="primary"
+              onClick={() => {
+                setIsLoadingTask(true);
+                removeTaskFromUser(props.task);
+              }}
+              disabled={isLoadingTask}
```

Comment:
This part of the code says when `isLoadingTask` is `true`, it renders `<p>Loading...</p>`, and when `isLoadingTask` is `false` it renders this button.

So on line 51 `disabled={isLoadingTask}` would always be `disabled={false}`, no?